packages/disml/disml.0.2.4/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "Adelyn Breedlove <tenseiken595@gmail.com>"
+authors: "Adelyn Breedlove <tenseiken595@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/Mishio595/disml"
+doc: "https://mishio595.gitlab.io/disml/"
+dev-repo: "git+https://gitlab.com/Mishio595/disml"
+bug-reports: "https://gitlab.com/Mishio595/disml/issues"
+tags: ["discord"]
+synopsis: "An OCaml library for interfacing with the Discord API"
+description: """
+Dis.ml is a library that provides a high-level interface to the Discord API.
+Key features include:
+* Automatic sharding
+* Deserialization of Discord objects to record types with related helper methods
+* Automatic rate-limiting
+
+For examples, see `/bin` in the git repo.
+"""
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {build & >= "1.3.0"}
+  "async_ssl" {>= "v0.11.0"}
+  "core" {>= "v0.11.3"}
+  "cohttp-async" {>= "1.2.0"}
+  "ppx_sexp_conv" {>= "v0.11.2"}
+  "ppx_deriving_yojson" {>= "3.1"}
+  "websocket-async" {>= "2.12"}
+  "decompress"

```

Comment:
is there no lower-bound constraints for decompress. The API changed a lot between versions and I'm worried it wouldn't compile with earlier versions. Have you tried earlier versions?